src/test/java/hu/dpc/rt/kafkastreamer/importer/TestUtil.java

```diff
@@ -0,0 +1,14 @@
+package hu.dpc.rt.kafkastreamer.importer;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+public class TestUtil {
+
+    @Test
+    public void test1() {
+        System.out.println(UUID.randomUUID().toString());
```

Comment:
Is the test and JUnit setup really required? It's pretty much dead code.

Reply:
I know what you mean. I added the `useJUnitPlatform` because otherwise, if you do add tests, they are never run. So I added one to be able to verify locally that it will be run (using --info). I don't have much opinion on the rest, only that tests would be nice to have

Reply:
Gonna have a look for some easy targets to kick off tests

Reply:
Hm, understandable and in future we should have tests. Well, since there are tests as of now, let's keep JUnit dependency out for now. What you think?